src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2565,6 +2569,32 @@ This package will be deleted after the next Dynamo restart.</value>
   <data name="PackageContextMenuUnmarkUnloadPackageTooltip" xml:space="preserve">
     <value>Remove the scheduled unload status</value>
   </data>
+  <data name="PackageViewContextMenuLoadText">
+    <value>Load</value>
+  </data>
+  <data name="PackageViewContextMenuLoadTooltip">
+    <value>Load this package into Dynamo. Other packages with the same name will be automatically deleted</value>
+  </data>
+  <data name="PackageViewContextMenuUnmarkForLoadText">
+     <value>Cancel Load</value>
+  </data>
+  <data name="PackageViewContextMenuUnmarkForLoadTooltip">
+     <value>Remove scheduled load status and all schduled for delete statuses from all conflicting packages</value>
+  </data>
+  <data name="MessageLoadBuiltInPackageWithRestart" xml:space="preserve">
+    <value>The built in package {1} cannot be loaded.
+Loading it would conflict with one or more packages that are already loaded. 
+To load {1}, Dynamo needs to first delete the conflicting packages. 
+Restart Dynamo to complete the install.
+
+Uninstall the following packages: {0}?</value>
+  </data>
+  <data name="MessageLoadBuiltInPackage" xml:space="preserve">
+    <value>Loading the built-in package {1} conflicts with one or more packages that are already loadeded. 
```

Comment:
typo: loaded

Reply:
is this for non-assembly containing packages that are conflicting?

Reply:
YEs...which pretty much translates to "Loading that does not require a restart".